core/src/variant.rs

```diff
@@ -775,6 +776,10 @@ godot_uint_impl!(u8);
 godot_uint_impl!(u16);
 godot_uint_impl!(u32);
 godot_uint_impl!(u64);
+godot_uint_impl!(usize);
+use std::ffi::c_void;
+godot_uint_impl!(*mut c_void);
+godot_uint_impl!(*const c_void);
```

Comment:
I don't think I'm a fan of explicitly allowing to store raw pointers in `Variant`s, is there a specific use case you need this for?

Reply:
Yes. I want to store data structure in rust in Godot without implementing `ToVariant`, which is hard to do. I still need to use `unsafe` keyword.